tickets/src/main/java/no/unit/nva/publication/ticket/create/TicketResolver.java

```diff
@@ -114,37 +113,11 @@ private boolean isPublishingRequestCase(TicketEntry ticket) {
         return ticket instanceof PublishingRequestCase;
     }
 
-    private TicketEntry persistTicket(TicketEntry newTicket) throws ApiGatewayException {
+    private TicketEntry persistTicket(TicketEntry newTicket) {
         return attempt(() -> newTicket.persistNewTicket(ticketService))
-                   .orElse(fail -> handleCreationException(fail.getException(), newTicket));
-    }
-
-    private TicketEntry updateAlreadyExistingTicket(TicketEntry newTicket) {
-        var customerId = newTicket.getCustomerId();
-        var resourceIdentifier = newTicket.getResourceIdentifier();
-        return ticketService.fetchTicketByResourceIdentifier(customerId, resourceIdentifier, newTicket.getClass())
-                   .map(this::updateTicket)
                    .orElseThrow();
     }
 
-    private TicketEntry updateTicket(TicketEntry ticket) {
-        ticket.persistUpdate(ticketService);
-        return ticket;
-    }
-
-    private TicketEntry handleCreationException(Exception exception, TicketEntry newTicket) throws ApiGatewayException {
-        if (exception instanceof TransactionFailedException) {
-            return updateAlreadyExistingTicket(newTicket);
-        }
-        if (exception instanceof RuntimeException) {
-            throw (RuntimeException) exception;
-        }
-        if (exception instanceof ApiGatewayException) {
-            throw (ApiGatewayException) exception;
-        }
-        throw new RuntimeException(exception);
-    }
-
```

Comment:
This piece of code is not tested at all. I guess it not needed, cause TicketResolver is used in CreateTicketHandler. Why would we update existing ticket when attempting to create new one? DoiRequest? From now on it should be possible to persist multiple DoiRequests